src/python/turicreate/toolkits/sound_classifier/_tf_sound_classifier.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2019 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+from turicreate.util import _ProgressTablePrinter
+import tensorflow as _tf
+import numpy as _np
+import time as _time
+from .._tf_model import TensorFlowModel
+import turicreate.toolkits._tf_utils as _utils
+
+import tensorflow.compat.v1 as _tf
+_tf.disable_v2_behavior()
+
+
+class SoundClassifierTensorFlowModel(TensorFlowModel):
+
+    def __init__(self, batch_size, num_classes):
+        """
+        Defines the TensorFlow model, loss, optimisation and accuracy.
+
+        """
+        #for key in net_params.keys():
+        #    net_params[key] = _utils.convert_shared_float_array_to_numpy(net_params[key])
+
+        _tf.reset_default_graph()
+
+        self.num_classes = num_classes
+        self.batch_size = batch_size
+
+        self.x = _tf.placeholder("float", [None, 1, 12288])
+        self.y = _tf.placeholder("float", [None, self.num_classes])
+
+        # Weights
+        weights = {
+        'sound_dense0_weight'  : _tf.Variable(_tf.random.uniform([12288, 100]), name='sound_dense0_weight'),
```

Comment:
Are these weights going to be overwritten by MXNet? I realized when doing style transfer tensorflow graph that argument `initial_value` is extremely useful.

Reply:
That is a part I'm still figuring out how I want to do this. Since we are removing MXNET pieces I didn't want to rely on getting weights from MXNET